qemu-vanilla/debian.rules-template

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/make -f
+include /usr/share/dpkg/default.mk
+export LANG=C
+%:
+	dh $@
+
+override_dh_auto_configure:
+	./configure \
+	--disable-bluez \
```

Comment:
Same comment - this should be calling https://github.com/kata-containers/packaging/blob/master/scripts/configure-hypervisor.sh so we can guarantee debs+rpms are consistent and using the same options as other repos that need to build the hypervisor.